plugins/Marketplace/resources/qml/InstallMissingPackagesDialog.qml

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) 2021 Ultimaker B.V.
```

Comment:
Currently, this QML file is largely a copy of `Marketplace.qml`. I'd prefer it if we had a base-class we could use for both.

That said, it could be that the designs diverge, in that case it may be best to decide to keep the copy for now.

In any case, I want to make sure that it's a conscious decision on our side, and not 'just' copy-pasta.

Reply:
We didn't choose for this option as this forces us  to introduce some very specific properties in the Marketplace in order to use the component in both this instance and in the Marketplace itself. This very specific properties can be a pain to maintain as it is not immediately clear why they are needed.

However after testing the suggested approach I do agree that the removal of the code duplication outweighs the specific properties in the Marketplace component.

https://github.com/Ultimaker/Cura/pull/12434/commits/bee4da63681cb38c25b593456409fe6cf8a50777